crates/etl/src/lib.rs

```diff
@@ -99,7 +100,7 @@ where
 
     fn flush(&mut self) {
         self.buffer_size_bytes = 0;
-        self.buffer.sort_unstable_by(|a, b| a.0.cmp(&b.0));
+        self.buffer.par_sort_unstable_by(|a, b| a.0.cmp(&b.0));
```

Comment:
wo large to we expect this to be?

we might not benefit from par sort here if not large, can do a length check and use par_sort if large if small buffers are possible

Reply:
given its only used on the pipeline, i'd say pretty large. Max at 100MB for headers, and 500MB worth for txlookup